packages/datadog_flutter_plugin/android/src/main/kotlin/com/datadoghq/flutter/DatadogSdkPlugin.kt

```diff
@@ -251,17 +253,23 @@ class DatadogSdkPlugin : FlutterPlugin, MethodCallHandler {
         return builder
     }
 
-//    private fun attachToExisting(): Map<String, Any> {
-//        var rumEnabled = false
-//        if (GlobalRum.isRegistered()) {
-//            rumEnabled = true
-//            rumPlugin.attachToExistingSdk()
-//        }
-//
-//        return mapOf<String, Any>(
-//            "rumEnabled" to rumEnabled
-//        )
-//    }
+    private fun attachToExisting(): Map<String, Any> {
+        var loggingEnabled = false
+        if (Logs.isEnabled()) {
+            loggingEnabled = true
+        }
```

Comment:
it should be possible just to write `val loggingEnabled = Logs.isEnabled()`?

Reply:
Yes. I thought I needed more code here like in RUM.